### PR TITLE
Joystick: Update to match joystick page in QGC 4.0

### PR DESCRIPTION
**Pull request by Williangalvani**
I am not sure I got the "Button Frequency" function right. Feel free to correct me

### CODE REVIEW

File: en/SetupView/Joystick.md [hunk 1]
```diff
@@ -30,12 +30,17 @@ To configure a joystick:
 
    ![Joystick setup - PlayStation](../../assets/setup/joystick_sony_playstation.jpg)
 1. Make sure your joystick is selected in the **Active joystick** dropdown.
-1. Press the **Calibrate** button and then follow the on-screen instructions to calibrate/move the sticks.
-1. Test the buttons and sticks work as intended by pressing them, and viewing the result in the Axis/Button monitor.
+1. Go to the **Calibrate** Tab, press the **Start** button and then follow the on-screen instructions to calibrate/move the sticks. The joystick is enabled as the last step of the calibration process.
+   ![Joystick setup - Calibration](../../assets/setup/joystick_calibration.jpg)
+1. Test the buttons and sticks work as intended by pressing them, and viewing the result in the Axis/Button monitor in the **General** tab.
 1. Select the flight modes/vehicle functions activated by each joystick button.
-1. Check the **Enable joystick input** checkbox to begin sending joystick commands to the vehicle.
```
Comment (hamishwillee) [inline en/SetupView/Joystick.md]: So joystick commands are now always/automatically sent when the joystick is connected?

File: en/SetupView/Joystick.md [hunk 2]
```diff
@@ -30,12 +30,17 @@ To configure a joystick:
 
    ![Joystick setup - PlayStation](../../assets/setup/joystick_sony_playstation.jpg)
```
Comment (hamishwillee) [inline en/SetupView/Joystick.md]: Why is this file deleted in the list above?

**Comment by hamishwillee** [comment]
@DonLakeFlyer I've added some comments, but I don't know what functionality was changed, so would be good for you to review also.

File: en/SetupView/Joystick.md (hunk 1 above)
Comment (Williangalvani) [reply en/SetupView/Joystick.md]: They are enabled as the last step of the calibration, check line 33.

File: en/SetupView/Joystick.md (hunk 2 above)
Comment (Williangalvani) [reply en/SetupView/Joystick.md]: Do you mean the files list? It is not deleted. I think it is in red because the file size shrunk

Comment (hamishwillee) [reply en/SetupView/Joystick.md]: Yes I did. Good point. Thank you.

File: en/SetupView/Joystick.md (hunk 1 above)
Comment (hamishwillee) [reply en/SetupView/Joystick.md]: I see that now. It wasn't obvious. Do you think we should perhaps make that a note?

Comment (Williangalvani) [reply en/SetupView/Joystick.md]: Sure, feel free to add it!

Comment (hamishwillee) [reply en/SetupView/Joystick.md]: Done! This PR now just waiting on you @DonLakeFlyer

Comment (DonLakeFlyer) [reply en/SetupView/Joystick.md]: In reality I haven't looked at Joystick config in ages. @dogmaphobic Was the one would did the latest rework of the ui. So he'd be better to review.

Comment (dogmaphobic) [reply en/SetupView/Joystick.md]: It all seems fine with the exception of:

 **Enable Gimbal Control**: Enabled two additional channels for controlling a gimbal.

This is not enabled (internally there is no code to handle it). The joystick calibration and setup is too hardcoded for 4 attitude axis only. That needs some work before we can introduce more than 4 axis.

Comment (hamishwillee) [reply en/SetupView/Joystick.md]: @dogmaphobic Thanks. So that is what the UI shows - do we just add a note "not supported" or similar below that?

Comment (DonLakeFlyer) [reply en/SetupView/Joystick.md]: There is a checkbox for something which you can check but does nothing?

Comment (hamishwillee) [reply en/SetupView/Joystick.md]: That's what it sounds like!

**Comment by Williangalvani** [comment]
can someone merge this?
